server/app/views/questiontypes/AddressQuestionRenderer.java

```diff
@@ -107,10 +117,76 @@ protected DivTag renderInputTags(
                 addressOptionalField.getInputTag(),
                 /** Third line of address entry: City, State, Zip */
                 div()
-                    .withClasses("grid", "grid-cols-3", "gap-2")
+                    .withClasses("grid", "grid-cols-3", "gap-3")
                     .with(
-                        cityField.getInputTag(), stateField.getInputTag(), zipField.getInputTag()));
+                        cityField.getInputTag(),
+                        stateField.getSelectTag(),
```

Comment:
Add a TODO here with the issue you made to fix the styling

Reply:
Done!